sdk/react/ooyalaSkinBridgeListener.js

```diff
@@ -1,18 +1,31 @@
+// @flow
+import { Platform } from 'react-native';
+
+import { CONTENT_TYPES, OVERLAY_TYPES, SCREEN_TYPES } from './constants';
+import Log from './log';
+import type { Marker } from './src/types/Marker';
+
 /**
- * The OoyalaSkinBridgeListener handles all of the listening of Player events from the Bridge
+ * The OoyalaSkinBridgeListener handles all of the listening of Player events from the Bridge.
  */
-import {
-  Platform
-} from 'react-native';
-
-import {
-  CONTENT_TYPES,
-  SCREEN_TYPES,
-  OVERLAY_TYPES,
-} from './constants';
-import Log from './log';
+export default class OoyalaSkinBridgeListener {
+  static parseMarkers(markers: ?Array<string>): Array<Marker> {
```

Comment:
I'm not sure that it is the best place for this method. What do you think about move it somewhere?

Reply:
I believe we have to parse markers into the array of JSONs when the bridge is getting it, but I agree the code of parsing can be placed somewhere else. Do you have suggestions?

Reply:
I'm not sure. Maybe we can create utils folder, move util into it and also create JsonParser class that will contains this method. Also in future we can add json parsers here.

Reply:
JSON parser here is just a built-in global JSON.parse method, we don't need to place it anywhere. Still, the markers array parsing logic is specific to markers, so we can create such util folder inside `src/markers/`, what do you think?

Reply:
Yes, It sounds good for me.

Reply:
https://github.com/ooyala/native-skin/pull/863/files#diff-a7855b4e985cd6026a8aedd6d6c7b80b